ReportService.Tests/DbExporterTests.cs

```diff
@@ -0,0 +1,167 @@
+using Autofac;
+using AutoMapper;
+using Monik.Client;
+using Monik.Common;
+using Moq;
+using ReportService.Core;
+using ReportService.Entities;
+using ReportService.Entities.Dto;
+using ReportService.Extensions;
+using ReportService.Interfaces.Core;
+using ReportService.Interfaces.Operations;
+using ReportService.Interfaces.ReportTask;
+using ReportService.Operations.DataExporters;
+using ReportService.Operations.DataExporters.Configurations;
+using ReportService.Operations.DataExporters.Dependencies;
+using ReportService.ReportTask;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ReportService.Tests
+{
+
+    public class DbExporterTests
+    {
+
+        #region fields
+
+        private ILifetimeScope autofac;
+        private Mock<IMonik> monikMock;
+        private readonly ContainerBuilder builder;
+        #endregion
+
+
+        #region ctor
+
+        public DbExporterTests()
+        {
+            builder = new ContainerBuilder();
+            ConfigureMapper(builder);
+        }
+        #endregion
+
+
+        #region testing methods
+
+        [Fact]
+        public async void TestExceptionMessageOnFailedDbStructureCheck()
+        {
+            builder.RegisterType<B2BExporter>();
```

Comment:
check if it is possible to use WebApplicationFactory (https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.testing.webapplicationfactory-1?view=aspnetcore-3.0)